src/common/telemetryHelper.ts

```diff
@@ -32,8 +32,8 @@ export class TelemetryHelper {
         return { value: String(propertyValue), isPii: pii || false };
     }
 
-    public static addReactNativeVersionToEventProperties(reactNativeVersion: string, properties: ICommandTelemetryProperties = {}): any {
-        properties.reactNativeVersion = {
+    public static addReactNativeVersionToEventProperties(reactNativeVersion: string, properties: ICommandTelemetryProperties = {}, reactNativeVersionPropName = "reactNativeVersion"): any {

```

Comment:
Are you sure that we need autointialization here? What if this function will be used incorrectly and the property name will be missed in cases of some other packages?

Reply:
If this function is used incorrectly, it will add `reactNativeVersion` property, so I'll change the 'reactNativeVersionPropName` parameter from optional to mandatory.